### ISSUE TITLE
NVDA+Firefox cannot select SVG toggleable elements

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#62]
This is an issue in reference to #23 which is related to the **"primitives and nested role=button"** branch.

<img width="811" alt="Screenshot of NVDA in test environment. A blue circle is highlighting Text Viewer text that reads '67 file fragments have been loaded from our record pressed  Records are now closed not pressed 
 now closed' while another blue circle is highlighting that only one element is selectable." src="https://user-images.githubusercontent.com/98043076/161095455-3391515c-386b-4b1f-99f1-709ffe178071.png">

### Issue Summary
NVDA can only select one toggleable child element, "5 Rect." The other child elements cannot be navigated to or selected individually (circle, path, text).  We would expect 7 navigable elements, and 4 toggleable elements. (This is connected to #61 issue.)

### Video of issue 
Example starts at 00:00:36
https://user-images.githubusercontent.com/98043076/160902176-489f09af-32fb-4cf7-b75e-b61945fe302e.mp4

### Steps to reproduce
Starting at the "Interactions Instructions" heading: Use NVDA down arrow key to navigate into the SVG space. Press the down arrow key twice.

### Expected behavior (Pass/Fail)
- **FAIL** We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- **FAIL** We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- **FAIL** We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating.
- **FAIL** The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- **FAIL** The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- **FAIL** The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- **FAIL** Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- **Pass** We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- **Pass** The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed visually in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- **FAIL** We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- **FAIL** We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).
- **???** We expect that if the main roles are not used, the fallback roles will be (if they are enabled). We are not sure how to design for this initially, but are mentioning it as important.
- **Pass** We expect that a SR will be able to toggle the element on or off.
- **Pass** We expect that a SR will announce the toggle state correctly when navigated to.
- **Pass** We expect that a SR will announce the toggle state correctly when changed.
- **Pass** We expect that the aria live region for each test type (HTML and SVG) will correctly announce that something has happened with each toggle.

### Technical Details
NVDA 2021.3.4
Firefox 98.0.2
Windows 11 Build 22000.556
